### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by bzwartsenberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,79 @@
...
+
+
+## Scenario overview
+TorchDriveEnv comes bundled with a predefined collection of diverse and challenging training and testing scenarios. Training scenarios were created using the InvertedAI API, while the testing scenarios were hand tailored for interesting challenges. An overview of the testing scenarios is given in the visualization below. Note that while the ego agent has fixed waypoints in these scenarios, its starting position is sampled uniformly between the first two waypoints, and NPCs are initialized dynamically using the InvertedAI API. This results in a large variation in the available scenarios. 
+
+<table>
+  <tr>
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/ab82ec1e-fe79-4721-a996-512162032894.png" alt="Three Way"><br>Three Way</td>
```
Comment (zhangk1551) [inline README.md]: The sizes of the images here are slightly different. Could you try:
`` 
to resize the images here?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,79 @@
...
+To use the NPC's provided through the InvertedAI API, provide a valid API key through the environment variable `IAI_API_KEY`.
+
+## Wandb
+TorchDriveEnv is integrated with Weights & Biases for experiment logging, to make use of it provide an API key through `WANDB_API_KEY`.
+
+## Running experiments
+The results reported in the paper can be reproduced by cloning this repository and running:
+
+``` 
+python rl_training.py --config_file=env_configs/multi_agent/sac_training.yml
```
Comment (zhangk1551) [inline README.md:68]: The usage here might be outdated. The config files are refactored into: https://github.com/inverted-ai/torchdriveenv/blob/95fe6ff3c5e3036d0b837ac71d149c0b3d2448a1/examples/env_configs/rl_training.yml

File: README.md [hunk 3]
```diff
@@ -1,32 +1,79 @@
...
+The reward for TorchDriveEnv is based on waypoints that are provided to the ego agent, as well as a reward for movement and smoothness. Episodes that cause infractions (causing collisions or going offroad) are terminated early, otherwise the episode is terminated after a fixed number of steps. For a full overview of the reward, refer to the paper linked below. 
+
 # Installation
 
-The basic installation of torchdriveenv uses an OpenCV renderer, which is slower but easy to install. PyTorch3D renderer can be faster, but it requires specific versions of CUDA and PyTorch, so it is best installed in Docker.
+The basic installation of torchdriveenv uses an OpenCV renderer, which is slower but easy to install. PyTorch3D renderer can be faster, but it requires specific versions of CUDA and PyTorch, so it is best installed using Docker.
+
+TorchDriveEnv comes with an example that integrates [Stable-Baselines3](https://stable-baselines3.readthedocs.io/en/master), which can be used for testing purposes.
 
 ## Opencv rendering
```
Comment (zhangk1551) [inline README.md]: `OpenCV` would be more accurate.

**Review by zhangk1551 — COMMENTED** [review]
Thanks for making the readme more user-friendly!

File: README.md (hunk 3 above)
Comment (bzwartsenberg) [reply README.md]: Is this a comment about how to stylize it? I.e. `OpenCV` vs OpenCV?

File: README.md (hunk 2 above)
Comment (bzwartsenberg) [reply README.md:68]: Wilder updated this in his last PR, so that users can run all the experiments in the paper without having to modify any files.

File: README.md (hunk 1 above)
Comment (bzwartsenberg) [reply README.md]: Thank you! I was trying to figure out how to do this, but I was not able to figure this out. My concern is that if I set it to 100, it will always be a fixed size, which will look strange at large or small resolutions. Do you know any way to set it to the size of the table cell?
